### PR TITLE
Mes 725 iterate dl25

### PR DESCRIPTION
**Pull request by ataylorBJSS**
- Integrate DL25 v2 prototype into the e2e journey
- update store functionality (resetFault)
- Update manoeuvres section (and make eco complete btn consistent)
- darken section borders
- Try to align columns better
- removed undo last fault elements/functionality, as this is no longer used.

screen shot 2018-05-03 at 15 04 08
screen shot 2018-05-03 at 15 03 51

### CODE REVIEW

File: src/components/all-on-one-form-eta-button/all-on-one-form-eta-button.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ion-row align-items-center no-padding [ngClass]="{'active' : isEnabled, 'pAction': pActive, 'vAction': vActive }">
```
Comment (danielchildsbjss) [inline src/components/all-on-one-form-eta-button/all-on-one-form-eta-button.html]: pAction, vAction?

File: src/providers/fault-store/fault-store.ts [hunk 2]
```diff
@@ -151,15 +136,18 @@ export class FaultStoreProvider {
...
 
   calculateTestResult() {
-    if (this.drivingFaultsNumber >= 16 || this.seriousFaultsNumber > 0 || this.dangerousFaultsNumber > 0) {
-      return this.testResult = TestResult.Fail;
+    if (
+      this.drivingFaultsNumber >= 16 ||
+      this.seriousFaultsNumber > 0 ||
+      this.dangerousFaultsNumber > 0
+    ) {
+      return (this.testResult = TestResult.Fail);
```
Comment (danielchildsbjss) [inline src/providers/fault-store/fault-store.ts]: Can we change this to two lines with an assignment and a return? It looks like a comparison done wrong!

File: src/providers/fault-store/fault-store.ts [hunk 3]
```diff
@@ -151,15 +136,18 @@ export class FaultStoreProvider {
...
     }
-    return this.testResult = TestResult.Pass;
+    return (this.testResult = TestResult.Pass);
```
Comment (danielchildsbjss) [inline src/providers/fault-store/fault-store.ts]: Here too

File: src/components/all-on-one-form-eta-button/all-on-one-form-eta-button.html (hunk 1 above)
Comment (ataylorBJSS) [reply src/components/all-on-one-form-eta-button/all-on-one-form-eta-button.html]: If a physical action or a verbal action took place, add class to show sudo element p or v in button. Could probably be worded better.

**Pull request merged**